test/local/account_routes.js

```diff
@@ -161,6 +165,104 @@ test(
 )
 
 test(
+  '/recovery_email/status with sign-in confirmation enabled',
+  function (t) {
+
+    var configOptions = {
+      signinConfirmation: {
+        enabled: true,
+        sample_rate: 1.0
+      }
+    }
+
+    var mockDB = {
+      deleteAccount: sinon.spy()
+    }
+
+    var mockRequest = {
+      auth: {
+        credentials: {
+          uid: uuid.v4('binary').toString('hex'),
+          email: TEST_EMAIL,
+          emailVerified: true,
+          tokenVerified: true
```

Comment:
It would be interesting to add one of these tests with `emailVerified: true` but `tokenVerified: false`, to confirm that it reflects the expected behaviour.